tests/core/conversion/converters/test_select.cpp

```diff
@@ -365,6 +365,208 @@ TEST(Converters, ATenSliceNegEndIndexConvertsCorrectly) {
   ASSERT_TRUE(torch_tensorrt::tests::util::almostEqual(jit_results[0], trt, 2e-6));
 }
 
+TEST(Converters, ATenSliceDynamicBatchConvertsCorrectly) {
+  const auto graph = R"IR(
+        graph(%x.1 : Tensor):
+              %2 : None = prim::Constant()
+              %dim : int = prim::Constant[value=0]()
+              %start : int = prim::Constant[value=1]()
+              %end : int = prim::Constant[value=15]()
+              %step : int = prim::Constant[value=2]()
+              %9 : Tensor = aten::slice(%x.1, %dim, %start, %end, %step)
+              return (%9))IR";
+
+  auto g = std::make_shared<torch::jit::Graph>();
+
+  torch::jit::parseIR(graph, g.get());
+
+  auto in = at::randint(1, 10, {16, 32}, {at::kCUDA});
+
+  auto jit_in = at::clone(in);
+  auto params = torch_tensorrt::core::ir::get_static_params(g->inputs(), {});
+  auto jit_results = torch_tensorrt::tests::util::RunGraph(g, params, {jit_in});
+
+  auto trt_in = at::clone(in);
+  // dynamic shape in batch
+  auto trt_results = torch_tensorrt::tests::util::RunGraphEngineDynamic(g, params, {trt_in}, true);
+  auto trt = trt_results[0].reshape(jit_results[0].sizes());
+
+  ASSERT_TRUE(torch_tensorrt::tests::util::almostEqual(jit_results[0], trt, 2e-6));
+}
+
+TEST(Converters, ATenSliceDynamicBatchLargeEndConvertsCorrectly) {
+  const auto graph = R"IR(
+        graph(%x.1 : Tensor):
+              %2 : None = prim::Constant()
+              %dim : int = prim::Constant[value=0]()
+              %start : int = prim::Constant[value=1]()
+              %end : int = prim::Constant[value=99999]()
```

Comment:
I think you may need to handle the case where 'end' defaults to int64::max. I did some initial testing with your change and hit 'RuntimeError: value cannot be converted to type int without overflow'

https://github.com/pytorch/pytorch/blob/7f8e852dff54d073a079f94e2b7d271631c639ee/torch/csrc/jit/runtime/static/native_ops.cpp#L377

Reply:
fixed by latest commit

Reply:
I'm still seeing a failure for this model with your most recent changes:

```
class SliceTest(torch.nn.Module):
    def __init__(self):
        super(SliceTest, self).__init__()

    def forward(self, x):
        return (x[:, 0] + x[:, 2] + x[:, 1])
```

with this compile call:
```
trt_model = torch_tensorrt.compile(traced_cell.eval().cuda(), inputs=[
        torch_tensorrt.Input(
                min_shape=[1, 3],
                opt_shape=[32, 3],
                max_shape=[64, 3],
                dtype=torch.float32
        ),
    ],  **{
        "truncate_long_and_double": True,
        "min_block_size": 1
    })
```

Error:
`Error Code 4: Internal Error ((Unnamed Layer* 1) [Slice]_output: tensor volume exceeds (2^31)-1, dimensions are [2147483647,3])`

Reply:
I am using this commit to test the code https://github.com/pytorch/TensorRT/pull/1110/commits/979d9d1f3083fdb578e2d678dff471556192edef

Reply:
You can try the lastest commit now. This time it will work :)